src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,141 @@
+import { Text, View, Button, StyleSheet } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import { supabase } from '../utils/hooks/supabase';
+import { useAuthentication } from "../utils/hooks/useAuthentication";
+import {useState, useEffect} from 'react';
+
+export default function SettingsScreen() {
```

Comment:
Looks like this might be a duplicate? There is a SettingsScreen and a SettingScreen. Remove the one that isn't needed and test if works!